### PR TITLE
[KeyVault] updating package versions to republish and resign.

### PR DESCRIPTION
**Pull request by schaabs**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Core/Microsoft.Azure.KeyVault.Core.csproj [hunk 1]
```diff
@@ -5,11 +5,11 @@
     <Description>Microsoft Azure Key Vault Core Class Library</Description>
     <AssemblyTitle>Microsoft Azure Key Vault Core</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.KeyVault.Core</AssemblyName>
-    <VersionPrefix>3.0.0</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Core/Microsoft.Azure.KeyVault.Core.csproj]: Please replace `VersionPrefix` with `Version`

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography.Tests/Microsoft.Azure.KeyVault.Cryptography.Tests.csproj [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <Description>Microsoft Azure Key Vault Cryptography tests</Description>
     <AssemblyTitle>Microsoft Azure Key Vault Cryptography tests</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.KeyVault.Cryptography.Tests</AssemblyName>
-    <VersionPrefix>3.0.0-alpha</VersionPrefix>    
+    <VersionPrefix>3.0.1</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography.Tests/Microsoft.Azure.KeyVault.Cryptography.Tests.csproj]: Please replace `VersionPrefix` with `Version`
Test projects are never publish, please set the version to `1.0.0` and leave it unchanged

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography.Tests/Microsoft.Azure.KeyVault.Cryptography.Tests.csproj [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <Description>Microsoft Azure Key Vault Cryptography tests</Description>
     <AssemblyTitle>Microsoft Azure Key Vault Cryptography tests</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.KeyVault.Cryptography.Tests</AssemblyName>
-    <VersionPrefix>3.0.0-alpha</VersionPrefix>    
+    <VersionPrefix>3.0.1</VersionPrefix>    
   </PropertyGroup>
   <!--<PropertyGroup>
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
```
Comment (dsgouda) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography.Tests/Microsoft.Azure.KeyVault.Cryptography.Tests.csproj]: Remove commented lines

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Microsoft.Azure.KeyVault.Cryptography.csproj [hunk 4]
```diff
@@ -4,14 +4,12 @@
     <PackageId>Microsoft.Azure.KeyVault.Cryptography</PackageId>
     <Description>Microsoft Azure Key Vault Cryptography Class Library</Description>
     <AssemblyTitle>Microsoft Azure Key Vault Cryptography</AssemblyTitle>
-    <VersionPrefix>3.0.0</VersionPrefix>
+    <VersionPrefix>3.0.1</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Microsoft.Azure.KeyVault.Cryptography.csproj]: `VersionPrefix`

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Extensions.Tests/Microsoft.Azure.KeyVault.Extensions.Tests.csproj [hunk 5]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.KeyVault.Extensions.Tests</PackageId>
     <Description>Microsoft Azure Key Vault Extensions tests</Description>
-    <VersionPrefix>3.0.0-alpha</VersionPrefix>
+    <VersionPrefix>3.0.1</VersionPrefix>
     <AssemblyName>Microsoft.Azure.KeyVault.Extensions.Tests</AssemblyName>    
   </PropertyGroup>
   <!--<PropertyGroup>
```
Comment (dsgouda) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Extensions.Tests/Microsoft.Azure.KeyVault.Extensions.Tests.csproj]: Commented lines

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Extensions.Tests/Microsoft.Azure.KeyVault.Extensions.Tests.csproj [hunk 6]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.KeyVault.Extensions.Tests</PackageId>
     <Description>Microsoft Azure Key Vault Extensions tests</Description>
-    <VersionPrefix>3.0.0-alpha</VersionPrefix>
+    <VersionPrefix>3.0.1</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Extensions.Tests/Microsoft.Azure.KeyVault.Extensions.Tests.csproj]: Test projects versioning must be `1.0.0`

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Extensions/Microsoft.Azure.KeyVault.Extensions.csproj [hunk 7]
```diff
@@ -5,10 +5,10 @@
     <Description>Microsoft Azure Key Vault Extensions Class Library</Description>
     <AssemblyTitle>Microsoft Azure Key Vault Extensions</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.KeyVault.Extensions</AssemblyName>
-    <VersionPrefix>3.0.0</VersionPrefix>
-    <PackageReleaseNotes>
+    <VersionPrefix>3.0.1</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Extensions/Microsoft.Azure.KeyVault.Extensions.csproj]: `VersionPrefix`

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.TestFramework/Microsoft.Azure.KeyVault.TestFramework.csproj [hunk 8]
```diff
@@ -4,7 +4,7 @@
     <Description>Microsoft Azure Key Vault Test Framework</Description>
     <AssemblyTitle>Microsoft Azure Key Vault Test Framework</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.KeyVault.TestFramework</AssemblyName>    
-    <VersionPrefix>3.0.0-alpha</VersionPrefix>    
+    <VersionPrefix>3.0.1</VersionPrefix>    
   </PropertyGroup>
   <!-- <PropertyGroup> -->
     <!-- <TargetFrameworks>netcoreapp1.1</TargetFrameworks> -->
```
Comment (dsgouda) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.TestFramework/Microsoft.Azure.KeyVault.TestFramework.csproj]: Same comments as above for test projects

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Tests/Microsoft.Azure.KeyVault.Tests.csproj [hunk 9]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.KeyVault.Tests</PackageId>
     <Description>Microsoft.Azure.KeyVault.Tests Class Library</Description>
-    <VersionPrefix>3.0.0-alpha</VersionPrefix>
+    <VersionPrefix>3.0.2</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Tests/Microsoft.Azure.KeyVault.Tests.csproj]: Same comments as above for test projects

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.WebKey.Tests/Microsoft.Azure.KeyVault.WebKey.Tests.csproj [hunk 10]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.KeyVault.WebKey.Tests</PackageId>
     <Description>Microsoft.Azure.KeyVault.WebKey.Tests Class Library</Description>
-    <VersionPrefix>3.0.0-alpha</VersionPrefix>
+    <VersionPrefix>3.0.2</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.WebKey.Tests/Microsoft.Azure.KeyVault.WebKey.Tests.csproj]: `VersionPrefix`

File: src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault/Microsoft.Azure.KeyVault.csproj [hunk 11]
```diff
@@ -4,10 +4,14 @@
...
     <AssemblyTitle>Microsoft Azure Key Vault</AssemblyTitle>
-    <VersionPrefix>3.0.1</VersionPrefix>
+    <VersionPrefix>3.0.2</VersionPrefix>
```
Comment (dsgouda) [inline src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault/Microsoft.Azure.KeyVault.csproj]: `VersionPrefix`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Left a few comments

**Comment by schaabs** [comment]
@dsgouda I believe I addressed all your comments.  The one not marked outdated is because the comment was on the removed line.  PTAL

**Review by dsgouda — APPROVED** [review]
LGTM
@schaabs Thanks for the cleanup

**Pull request merged**